zh/monitor-using-tidbmonitor.md

```diff
@@ -171,6 +171,99 @@ spec:
   imagePullPolicy: IfNotPresent
 ```
 
+## 开启 Ingress
+
+目前, `TidbMonitor` 提供了通过 Ingress 将 Prometheus/Grafana 服务暴露出去的方式，你可以通过[Ingress](https://kubernetes.io/zh/docs/concepts/services-networking/ingress/)了解更多关于 Ingress 的详情。
```

Comment:
Should we also mention that users need to deploy ingress controller by themselves?

Reply:
updated.